src/components/ui/WorkExperience/index.js

```diff
@@ -66,17 +61,18 @@ const renderEditor = ({ value: { experience, skills }, onValueChange }: EditorPr
 )
 
 const renderPresenter = ({ value: { experience }, activateEditingMode, isHover }: PresenterProps) => (
-  <div className={cx.presenterWrapper}>
-    <div className={cx.editGutter}>
-      {isHover && <PencilButton color='monochrome' onClick={activateEditingMode} />}
-    </div>
-    <WorkExperienceEntryPresenter experience={experience} />
-  </div>
+  <WorkExperienceEntryPresenter experience={experience}
+    headerDecoration={isHover && <PencilButton color='monochrome' onClick={activateEditingMode} />} />
```

Comment:
It's easier to read and edit when JSX syntax is better presented.

```suggestion
 <WorkExperienceEntryPresenter
    experience={experience}
    headerDecoration={isHover && (
      <PencilButton color='monochrome' onClick={activateEditingMode} />
    )}
  />
```

Reply:
Oops, sorry, looks like the suggestion block had wrong white-spaces for the indentation 😓 
```
  64:1  error  Expected indentation of 2 spaces but found 1
  64:2  error  Expected indentation of 2 space characters but found 1
  65:5  error  Expected indentation of 3 space characters but found 4
  66:5  error  Expected indentation of 3 space characters but found 4
```